### ISSUE TITLE
xs-security.json contains "[32m" and "[39m" when using compile --to xsuaa 

### ISSUE THREAD

**Issue by gregorwolf** [sapmentors/cap-community#47]
When I upgrade my example project [bookshop-nodejs](https://github.com/gregorwolf/bookshop-nodejs) to @sap/cds version 3.31.1:

```
cds -v
@sap/cds: 3.31.1
@sap/cds-compiler: 1.23.2
@sap/cds-foss: 1.1.0
@sap/cds-messaging: 1.7.0
@sap/cds-reflect: 2.10.1
@sap/cds-rest: 1.5.1
@sap/cds-services: 1.25.1
Node.js: v12.16.1
home: /Users/gwolf/Documents/Projects/cap/bookshop-nodejs/node_modules/@sap/cds
```
and then run the command:

`cds compile srv/ --to xsuaa  > xs-security.json`

the generated xs-security.json looks like this:

```json
{
  xsappname: [32m'bookshop-nodejs'[39m,
  [32m'tenant-mode'[39m: [32m'dedicated'[39m,
  scopes: [ { name: [32m'$XSAPPNAME.admin'[39m, description: [32m'admin'[39m } ],
  attributes: [],
  [32m'role-templates'[39m: [
    {
      name: [32m'admin'[39m,
      description: [32m'generated'[39m,
      [32m'scope-references'[39m: [ [32m'$XSAPPNAME.admin'[39m ],
      [32m'attribute-references'[39m: []
    }
  ]
}
```

with 3.21.3 it was generated like this:

```json
{
  "xsappname": "bookshop-nodejs",
  "tenant-mode": "dedicated",
  "scopes": [
    {
      "name": "$XSAPPNAME.admin",
      "description": "admin"
    }
  ],
  "attributes": [],
  "role-templates": [
    {
      "name": "admin",
      "description": "generated",
      "scope-references": [
        "$XSAPPNAME.admin"
      ],
      "attribute-references": []
    }
  ]
}
```

**Comment by wattnu** [comment]
I'm observing the very same for the transformation cds --> csn via the cds command.  For instance:
```
cds .\samples.cds > samples.csn
```
When opening with VS Code Editor (it's guessing this to be UTF-16 LE), this shows up like:
```
{
  namespace: [32m'my.domain'[39m,
  definitions: [Object: null prototype] {
    [32m'my.domain.Products'[39m: {
      kind: [32m'entity'[39m,
      elements: [Object: null prototype] {
        ID: { key: [33mtrue[39m, type: [32m'cds.Integer'[39m },
        name: { type: [32m'cds.String'[39m, length: [33m100[39m, notNull: [33mtrue[39m },
        stock: { type: [32m'cds.Integer'[39m },
...
``` 
When I'm looking just at the command line output of ``cds samples.cds`` (Powershell), then everything looks fine!

Opening samples.csn with Notepad++ shows the little squares as ``ESC``. So searching for common ESCAPE code (https://bluesock.org/~willkg/dev/ansi.html), I've found this to be color code Escape Sequences! Somehow, these are polluting the output ....

**Comment by gregorwolf** [comment]
Is fixed with @sap/cds version 3.31.2
